Gems/Prefab/gem.json

```diff
@@ -0,0 +1,12 @@
+{
+    "gem_name": "Prefab",
+    "display_name": "Prefab",
+    "license": "Apache-2.0 Or MIT",
+    "origin": "Open 3D Engine - o3de.org",
+    "type": "Code",
+    "summary": "Prefab Gem provides an Asset Processor module for prefabs, which are complex assets built by combining smaller entities. [Experimental] Enables a Prefab Relationship Viewer UI in the editor",
```

Comment:
For now, change the description to 
```
[Experimental] Enables a Prefab Relationship Viewer UI in the editor
```
since that's what it actually does right now.

The Prefab Gem **does** provide a builder for asset processor but its mandatory and cannot be turned off, so its not really worth showing in the GUI.

Reply:
Resolved